### PR TITLE
Update intro.md

### PR DESCRIPTION
**Pull request by rwieruch**
Hey there 👋 

Just recently I released a tutorial about React Testing Library which I wished to have when I started using RTL :) Kent [tweeted](https://twitter.com/kentcdodds/status/1270759772661243905) about it last week as well and I was blown away by the response! 

I thought maybe it's a great additional read for people reading the docs who prefer reading over watching. I intend to keep the tutorial up-to-date, because this will be my go-to guide if people in workshops ask me about testing in React. If userEvent makes it into the core library, I will update the tutorial ASAP 🚀  

Let me know what you think!

### CODE REVIEW

File: docs/react-testing-library/intro.md [hunk 1]
```diff
@@ -64,11 +64,13 @@ facilitate testing implementation details). Read more about this in
...
+## Tutorials
 
 Have a look at the "What is React Testing library?" video below for an
 introduction to the library.
 
 <a href="https://youtu.be/JKOwJUM4_RM">
   <img width="200px" alt="what is react testing library" src='https://img.youtube.com/vi/JKOwJUM4_RM/0.jpg' style="margin-left: 0" />
 </a>
+
+If you prefer reading over watching, check this [tutorial for React Testing Library](https://www.robinwieruch.de/react-testing-library).
```
Comment (kentcdodds) [inline docs/react-testing-library/intro.md:76]: Every video on https://TestingJavaScript.com has enhanced transcripts with inline images and code, so people can consume it via reading as well. So have about changing this line to:

```suggestion
Also, don't miss this [tutorial for React Testing Library](https://www.robinwieruch.de/react-testing-library).
```

Comment (rwieruch) [reply docs/react-testing-library/intro.md]: Ah sure! I just saw the video by @stolinski there and I wasn't sure whether he provides transcripts as well :) But I bet he has them too! 🚀

Comment (kentcdodds) [reply docs/react-testing-library/intro.md]: Oh 😅 I didn't look carefully enough and thought this was referring to my course 😅

**Comment by kentcdodds** [comment]
@all-contributors please add @rwieruch for tutorials

**Pull request merged**